### DISCUSSION TITLE
Look into supporting and transforming query parameters

### DISCUSSION THREAD

**Discussion by sebastiandedeyne** [spatie/laravel-export#89] [Ideas]
Query parameters can't be used in the export bundle, but we might be able to transform them to fully qualified URL's in the export.

This would be useful to use Laravel's default pagination and have it supported in the export.

Next.js solves this by supporting a path map: https://github.com/zeit/next.js/#static-html-export

This feature should be added at some point, but I don't think it's critical for a 1.0 release.

**Comment by acstll** [comment]
> Next.js solves this by supporting a path map: https://github.com/zeit/next.js/#static-html-export

New URL for reference: https://nextjs.org/docs/advanced-features/static-html-export

**Comment by booprodusan** [comment]
Hey!

Any updates on this?
